rocks-lib/src/lua/mod.rs

```diff
@@ -0,0 +1,70 @@
+use eyre::Result;
+use std::path::PathBuf;
+use target_lexicon::Triple;
+
+use crate::config::{Config, LuaVersion};
+
+pub struct Lua {
+    pub include_dir: PathBuf,
+    pub lib_dir: PathBuf,
+}
```

Comment:
suggestion: This might become confusing, as we also have `mlua::Lua`.

What if we call this `LuaLib` and the module `lua_lib`?

Reply:
Maybe `LuaInstallation` for verbosity's sake? `LuaLib` is still a tad ambiguous I'd say.

Reply:
Changed it to `lua_installation` but happy to shorten it down to e.g. `lua_install` if you prefer.
I don't mind longer names when they're nicely unambiguous like in this case.